Ports/wayland/package.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='wayland'
+version='1.21.0'
+useconfigure='true'
+configopts=(
+    "--buildtype=release"
+    "--cross-file=${SERENITY_BUILD_DIR}/meson-cross-file.txt"
+    "-Dlibraries=true"
+    "-Dscanner=true"
+    "-Dtests=false"
+    "-Ddocumentation=false"
+)
+depends=('expat' 'libxml2')
```

Comment:
```suggestion
depends=(
    'expat'
    'libxml2'
)
```

(produces nicer diffs/blame when adding more later, e.g. for wayland-server)